server/services/interventionsService.ts

```diff
@@ -51,42 +59,73 @@ export default class InterventionsService {
     return new RestClient('Interventions Service API Client', this.config, token)
   }
 
+  private convertError(error: unknown): InterventionsServiceError {
```

Comment:
how about `sanitiseError` as we're returning a sanitised error?

Reply:
I'm hesitant to call it that for a couple of reasons:

1. I don't really understand what the word "sanitise" means in the context of doing something with an error, nor what the existing `sanitiseError` method is quite meant to do. I don't really want to use a piece of vocabulary that I don't fully understand - I don't know whether what I'm doing is "sanitising" in the sense that the existing method understands that word.
2. This `convertError` method assumes that its input is already a `SanitisedError`, so I'm not sure whether you can "sanitise" something that's already "sanitised".

Reply:
Yeah, I guess "sanitise" might not be the right thing - I think my issue with `convertError` was, from what to what?  I wonder if there's a name that conveys that we're converting a form error, for example, to one of our well-crafted error types. I suppose I used the word `convert`, so I can see why you named it like you did! But unfortunately I can't think of an alternative this early in the day, sorry...

Reply:
Gonna lose the transformation aspect of the name and just call it `createServiceError` 😀